src/main/resources/assets/admin/common/js/ui/panel/SplitPanel.ts

```diff
@@ -327,20 +332,30 @@ export class SplitPanel
         this.splitter.hide();
     }
 
-    distribute() {
+    distribute(noResizeEvent: boolean = false): void {
         if (this.isHorizontal()) {
             this.firstPanel.getEl().setHeight(this.getPanelSizeString(1)).setWidth(null);
             this.secondPanel.getEl().setHeight(this.getPanelSizeString(2)).setWidth(null);
             this.splitter.getEl().setHeightPx(this.getSplitterThickness()).setWidth(null).setLeft(null);
             if (this.isVisible()) {
-                this.triggerResizeDebounced();
+                if (noResizeEvent) {
+                    this.firstPanelResponsiveItem.update();
+                    this.secondPanelResponsiveItem.update();
+                } else {
+                    this.triggerResizeDebounced();
+                }
             }
```

Comment:
This piece of code is duplicated 10 lines below. Extract to `doAfterDistribute` or something?

Reply:
Or maybe move this block after `if.. then..`, unless it has to be called before setting the splitter position.